server/plugin/graphql/lhs_request.go

```diff
@@ -95,21 +95,38 @@ func (c *Client) GetLHSData(ctx context.Context) ([]*github.Issue, []*github.Iss
 
 func getPR(prResp *prSearchNodes) *github.Issue {
 	resp := prResp.PullRequest
+	labels := getGithubLabels(resp.Labels.Nodes)
 
-	return getGithubIssue(resp.Number, resp.Title, resp.Author.Login, resp.Repository.URL, resp.URL, resp.CreatedAt, resp.UpdatedAt)
+	return getGithubIssue(resp.Number, resp.Title, resp.Author.Login, resp.Repository.URL, resp.URL, resp.CreatedAt, resp.UpdatedAt, labels, resp.Milestone.Title)
 }
 
 func getIssue(assignmentResp *assignmentSearchNodes) *github.Issue {
 	resp := assignmentResp.PullRequest
+	labels := getGithubLabels(resp.Labels.Nodes)
 
-	return getGithubIssue(resp.Number, resp.Title, resp.Author.Login, resp.Repository.URL, resp.URL, resp.CreatedAt, resp.UpdatedAt)
+	return getGithubIssue(resp.Number, resp.Title, resp.Author.Login, resp.Repository.URL, resp.URL, resp.CreatedAt, resp.UpdatedAt, labels, resp.Milestone.Title)
```

Comment:
Then we also have this `getIssue` function which is not clear how it differs from the `getGithubIssue` function on first glance. Maybe we can name this `getIssue` function to `newIssueFromAssignmentResponse`